### PR TITLE
Feature/strimzi context

### PR DESCRIPTION
**Pull request by arov00**

### CODE REVIEW

File: strimzi-operator/src/test/java/io/spoud/kcc/operator/DefaultTestProfile.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package io.spoud.kcc.operator;
+
+import io.quarkus.test.junit.QuarkusTestProfile;
+
+import java.util.Map;
+
+public class DefaultTestProfile implements QuarkusTestProfile {
+    @Override
+    public Map<String, String> getConfigOverrides() {
```
Comment (gaetancollaud) [inline strimzi-operator/src/test/java/io/spoud/kcc/operator/DefaultTestProfile.java:9]: why not just have an application.yaml in the test/resource folder ?

File: strimzi-operator/src/test/java/io/spoud/kcc/operator/OperatorTest.java [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+    }
+
+    private void delayedAsyncRun(ThrowableRunnable r) {
+        CompletableFuture.runAsync(() -> {
+            try {
+                Thread.sleep(1000); // give downstream code some time to prepare
+                r.run();
+            } catch (Exception e) {
+                e.printStackTrace();
+                fail(e);
```
Comment (gaetancollaud) [inline strimzi-operator/src/test/java/io/spoud/kcc/operator/OperatorTest.java:279]: nitpick I don't think you need to print the stack if you fail just after

```suggestion
                fail(e);
```

**Review by gaetancollaud — APPROVED** [review]
I really like the idea ! I was really confused by the folder `strimzi-operator`at first. Maybe we can use the full name of the operator directly.

File: strimzi-operator/src/test/java/io/spoud/kcc/operator/DefaultTestProfile.java (hunk 1 above)
Comment (arov00) [reply strimzi-operator/src/test/java/io/spoud/kcc/operator/DefaultTestProfile.java:9]: Good point, I don't think there's any reason to not use `application.yaml`. Will keep in mind for next time.

**Pull request merged**